iocore/net/UnixNet.cc

```diff
@@ -234,7 +234,7 @@ initialize_thread_for_net(EThread *thread)
   thread->schedule_every(inactivityCop, HRTIME_SECONDS(cop_freq));
 
   thread->set_tail_handler(nh);
-  thread->ep       = (EventIO *)ats_malloc(sizeof(EventIO));
+  thread->ep       = new EventIO();
```

Comment:
If call of the constructor is the point we should be able to do it like this.
```suggestion
  thread->ep       = (EventIO *)ats_malloc(sizeof(EventIO));
  new (thread->ep) EventIO();
```